client/json-rpc/src/lib.rs

```diff
@@ -13,3 +13,7 @@ pub use client::{
 pub use libra_json_rpc_types::{errors, views};
 pub use libra_types::{account_address::AccountAddress, transaction::SignedTransaction};
 pub use response::{JsonRpcResponse, ResponseAsView};
+
+// new implementation module
+
+pub mod async_client;
```

Comment:
Maybe collapse async_client/* into this dir? Or if you don't like that another suggestion would be to pub use the relevant types directly from here.

Reply:
I intend to make everything under async_client, it will be easier to fix later by search this word later after we finalize the rust-client codebase.

because some names may conflict if we direct pub in here, like JsonRpcResponse, 
in my mind, as long as it is a clean mod import, then next step is decide whether we move the whole async_client to a separate codebase, if so, we delete this one later and change the use path.
if we don't move to separate codebase, then we need figure out how to clean up legacy code in this module, maybe move all of them in to a legacy module and let everyone know (at least AOS uses this), or we move legacy code into separate codebase and ask people don't migrate to new client to use that instead.
Let me know what you think.